cli.js

```diff
@@ -212,41 +231,31 @@ const commands = {
       callback();
       return;
     }
-    cp.exec('ps -A | awk \'{if ($4 == "impress") print $1, $5}\'',
-      (err, stdout, stderr) => {
-        const error = err || stderr;
-        if (error) {
-          console.log(concolor.error(error.toString()));
-          callback();
-          return;
-        }
 
-        let processes = stdout.toString()
-          .split('\n')
-          .filter(line => line !== '')
-          .map(line => {
-            const parsedLine = line.split(' ');
-            return { pid: parsedLine[0], workerId: parsedLine[1] };
-          });
+    listImpressProcesses(force, (err, processes) => {
+      if (err) {
+        console.log(concolor.error(err.toString()));
```

Comment:
This is already being done inside the `listImpressProcesses` function.

Reply:
Actually, no. In `listImpressProcesses` I check if there was error and if there was one I pass it to callback, otherwise I pass `null` and `processes`. But that's a good call. I'll move log to `listImpressProcesses` so it won't be duplicated later in callback